python/ray/tune/ray_trial_executor.py

```diff
@@ -132,6 +138,11 @@ def cleanup(self, partial: bool = True):
                 logger.warning(
                     "Skipping cleanup - trainable.stop did not return in "
                     "time. Consider making `stop` a faster operation.")
+                if not partial and self._force_cleanup:
+                    logger.warning(
+                        "Forcing trainable cleanup by terminating actors.")
+                    self._cleanup_map = {}
```

Comment:
Does this actually stop the actors? Will this introduce dangling actors in remote nodes?